crypto/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //! The goal is provide a easier and less verbose API to create, manage and interact with MLS
 //! groups.
 #![doc = include_str!("../../README.md")]
-#![deny(missing_docs)]
+#![cfg_attr(not(test), deny(missing_docs))]
```

Comment:
Why add this now?

Reply:
No idea, removed it in [chore: fix docs](https://github.com/wireapp/core-crypto/pull/732/commits/fd93bce72ef8b8867e76b3f37466dc9cd8f30088).